.github/workflows/build.yml

```diff
@@ -133,6 +138,11 @@ jobs:
             python: "3.10"
             piparch: macosx_10_10_intel
 
+          - name: osx 3.11 intel
+            os: macos-latest
+            python: "3.11"
+            piparch: macosx_10_11_intel
```

Comment:
For my information.  What motivated your choosing `macosx_10_11_intel`?

Reply:
This was me misunderstanding the pattern in use - I saw `macosx_10_10_intel` for Python 3.10, just above these lines, and assumed the pattern should be followed.

Should this be changed to `macosx_10_9_intel`, or to `macosx_10_10_intel`?

Reply:
An unfortunate overlap of numbering.  eg. `10.10` refers to [OSX versions](https://en.wikipedia.org/wiki/OS_X_Yosemite).

I would suggest staying with `macosx_10_10_intel`.

Reply:
Updated in 67b398f08ffdef0b160be217f6af19e30a0a62b0

Reply:
> And if it's macosx_10_9_intel, should I also change the string for the Python 3.10 release?

Absent some complaint from an actual OSX user, as long as the tests pass I would leave things as they are.